requirements.txt

```diff
@@ -2,6 +2,8 @@ asgiref==3.5.2
 attrs==22.2.0
 black==22.12.0
 click==8.1.3
+colorama==0.4.6
+dj-database-url==1.2.0
```

Comment:
We use [dj-database-url](https://pypi.org/project/dj-database-url/) to parse our DATABASE_URL. Instead of writing each database configuration such DB_NAME, DB_USER, DB_HOST etc we can just write a DATABASE URL e.g postgresql://USER:PASSWORD@HOST:PORT/NAME then parse it using dj-database-url which will extract individual configurations.
It reduces the number of database environment variables to just one which is DATABASE_URL.